src/memory_pool_default.c

```diff
@@ -17,7 +17,7 @@
 umf_result_t umfPoolCreate(const umf_memory_pool_ops_t *ops,
```

Comment:
bump license dates, I guess 😉

Reply:
Ah yes, it's this time of year ;d Done.

Reply:
Btw, don't we have a check-license script here?

Reply:
nope, we don't